### DISCUSSION TITLE
Welcome to learn-k8s-from-scratch Discussions!

### DISCUSSION THREAD

**Discussion by xiaopeng163** [xiaopeng163/learn-k8s-from-scratch#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Matthiasbrat** [comment]
Hi, I saw https://learn-k8s-from-scratch.readthedocs.io/en/latest/index.html
is there any english version ?
If not, I'm willing to help with translations and improve the docs.
Also, I'm currently reading the entire kubernetes documentation v1.32 and am highlighting every important concept, I will then publish the entirety on github as "knowledge base" and wanting to organise everything using tags and categories. lmk if this sounds like something you'd want to help with.

**Reply by Matthiasbrat** [reply]
I'm motivated by the fact that I personnaly find kubernetes documentation really well thought and put together, but very hard to navigate.
- they have an api documentation (too straightforward)
- they have a a conceptual documentation (too many paragraphs)

For example, if you want to create a deployment, you have the api docs, but it doesn't explain anything. And if you want to read about deployments in the conceptual docs, you'll have to navigate to "concepts/workloads/workload management/deployments", where you find almost everything about deployments written in 1345 lines of markdown (way to much imo).

**Reply by Matthiasbrat** [reply]
**I am not rewriting the docs using my own words! I'm highlighting, removing, moving and categorizing paragraphs, sentences and examples.**
